foxy/distribution.yaml

```diff
@@ -853,6 +853,12 @@ repositories:
       url: https://github.com/stack-of-tasks/eiquadprog.git
       version: devel
     status: maintained
+  embree_vendor:
+    release:
+      tags:
+        release: release/foxy/{package}/{version}
+      url: https://github.com/OUXT-Polaris/embree_vendor-release.git
```

Comment:
I noticed that the [license in the package.xml](https://github.com/OUXT-Polaris/embree_vendor/blob/6e12d079e09b385ba6f1e1912912969c2b09833b/package.xml#L8) doesn't match the [license file](https://github.com/OUXT-Polaris/embree_vendor/blob/6e12d079e09b385ba6f1e1912912969c2b09833b/LICENSE). Should the LICENSE file be changed to Apache 2?

Reply:
Thank you.
I fixed this problem.
The license of this package is Apache2.0